### PR TITLE
Add a script to browse mode to toggle inclusion of layout tables

### PR DESCRIPTION
**Pull request by LeonarddeR**
### Link to issue number:
None

### Summary of the issue:
Some tables on the web, even though they have data, are treated as layout tables as they don't follow the convention that headers are required for a table to be treated as a data table. It is currently not possible to switch inclusion of layout tables on the fly while in browse mode.

### Description of how this pull request fixes the issue:
Adds an unbound script to browse mode to toggle layout tables. Even though this is a document formatting setting according to the config spec, I made this browse mode only due to the fact that from a UX perspectieve, this is a browse mode setting.

### Testing performed:
Toggled this setting while in a layout table in Firefox browse mode, detection changed on the fly as expected.

### Known issues with pull request:
None i'm aware of

### Change log entry:
* Changes
    + An unbound command has been added for browse mode to toggle the inclusion of layout tables on the fly. You can find this command in the Browse mode category of the Input Gestures dialog.

### CODE REVIEW

File: source/browseMode.py [hunk 1]
```diff
@@ -1760,6 +1760,19 @@ def script_previousColumn(self, gesture):
 	# Translators: the description for the previous table column script on browseMode documents.
 	script_previousColumn.__doc__ = _("moves to the previous table column")
 
+	def script_toggleIncludeLayoutTables(self,gesture):
+		if config.conf["documentFormatting"]["includeLayoutTables"]:
+			# Translators: The message announced when toggling the include layout tables document formatting setting.
+			state = _("include layout tables off")
```
Comment (derekriemer) [inline source/browseMode.py]: Maybe "layout tables off" or "layout tables disabled" might be more easy to understand.

File: source/browseMode.py [hunk 2]
```diff
@@ -1760,6 +1760,19 @@ def script_previousColumn(self, gesture):
...
+			# Translators: The message announced when toggling the include layout tables document formatting setting.
+			state = _("include layout tables off")
+			config.conf["documentFormatting"]["includeLayoutTables"]=False
+		else:
+			# Translators: The message announced when toggling the report tables document formatting setting.
+			state = _("include layout tables on")
+			config.conf["documentFormatting"]["includeLayoutTables"]=True
+		ui.message(state)
+	# Translators: Input help mode message for include layout tables command.
+	script_toggleIncludeLayoutTables.__doc__=_("Toggles on and off the inclusion of layout tables in browse mode")
```
Comment (derekriemer) [inline source/browseMode.py]: Do you need a category attribute here? I'm not familiar with this code.

**Comment by derekriemer** [comment]
Nice work!

**Review by derekriemer — APPROVED** [review]
Nice work!

File: source/browseMode.py (hunk 2 above)
Comment (leonardder) [reply source/browseMode.py]: Nope, the category is set on the class itself, so all the scripts have the category that the class prescribes.

File: source/browseMode.py (hunk 1 above)
Comment (leonardder) [reply source/browseMode.py]: I went the layout tables off route. Also fixed the translator comments in the process.

**Review by michaelDCurran — APPROVED** [review]

**Pull request merged**